modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/rest/resource/releases/SW360Release.java

```diff
@@ -129,19 +136,29 @@ public void setDownloadurl(String downloadurl) {
         this.downloadurl = downloadurl;
     }
 
-
     @JsonIgnore
-    public Map<String, String> getCoordinates() {
+    public Map<String, String> getPurls() {
         return externalIds.entrySet().stream()
-                .filter(e -> SW360CoordinateKeysToArtifactCoordinates.getValues().contains(e.getKey()))
+                .filter(e ->PURL_TYPE.stream()
+                .anyMatch(purlType -> e.getKey().equals(purlType)))
```

Comment:
Wrong indentation, that is part of the above filter, perhaps do not do it in a new line or do it as PURL_TYPE.contains(e.getKey())